src/adapters/postgres/user-adapter.ts

```
@@ -874,6 +909,30 @@ export class PostgresUserService implements IServicelocator {
+      const keycloakResponse = await getKeycloakAdminToken();
+      const token = keycloakResponse.data.access_token;
+
+      //Check user is exist in keycloakDB or not
+      const checkUserinKeyCloakandDb = await this.checkUserinKeyCloakandDb(updateField);
+      if (checkUserinKeyCloakandDb) {
+        return 'exists';
+      }
+
+      //Update user in keyCloakService
+      let updateResult = await updateUserInKeyCloak(updateField, token)
+      return updateResult ? true : false;
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)